living/swedish.md

```diff
@@ -19,6 +23,14 @@ Culture / History:
 Literature:
 - https://litteraturbanken.se/ (Literature Bank makes available large collection of fiction and other text)
 - https://www.boktugg.se/ (Blog about news, trends, authors, publishers and lot more about bookgs in general)
+- https://varldslitteratur.se/ (Wesbite with news and information about author biographies, translated works, poems and novels)
+- https://www.svelitt.se/index.html (Website of Swedish literature society that publishes yearbooks containing research on Swedish and other Nordic literature)
+- https://swedishbookreview.org/ (Home page of Swedish Book Review presenting Swedish-language literature to the English-speaking world, through feature articles, translated extracts, interviews, book reviews and more)
+- https://www.sls.fi/sv/ (Home page of SLS that preserves, explore and spread knowledge about Swedish culture in Finland )
+- https://www.mitrania.se/ (Literary newspaper for and about Fantasy, Science fiction and horror)
+- https://bokforlagetatlas.se/ (Atlas is a progressive publisher with its feet in the present, but with an eye on the future)
```

Comment:
Please change this description to something more neutral, such as:
> Swedish publishing house publishing fiction, non-fiction, and reportage

Reply:
Agreed and corrected